src/sanitizer.js

```diff
@@ -129,13 +139,18 @@ const WHITELISTED_ATTRS = [
   /* Attributes for amp-subscriptions. */
   'subscriptions-action',
   'subscriptions-actions',
-  'subscriptions-section',
+  'subscriptions-decorate',
+  'subscriptions-dialog',
   'subscriptions-display',
+  'subscriptions-section',
   'subscriptions-service',
-  'subscriptions-decorate',
 ];
 
-/** @const {!Object<string, !Array<string>>} */
+/**
+ * Tag-specific attribute whitelist for Caja. DOMPurify uses a tag-agnostic
+ * whitelist and custom attrs below are duped into PURIFY_CONFIG['ADD_ATTR'].
+ * @const {!Object<string, !Array<string>>}
+ */
```

Comment:
Some of these are valid in other contexts as well (e.g. rel) so I'd rather keep them as specific as possible. The tag specificity could be kept by whitelisting these in the attribute hook after checking the tagName.